frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -329,9 +332,13 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                                             <Col span={11} className="pl-4">
                                                 <RecentFeatureFlagInsights />
                                                 <div className="my-4" />
-                                                <FeatureFlagInstructions
-                                                    featureFlagKey={featureFlag.key || 'my-flag'}
-                                                />
+                                                {featureFlags[FEATURE_FLAGS.RECORDINGS_ON_FEATURE_FLAGS] ? (
+                                                    <FeatureFlagRecordings flagKey={featureFlag.key || 'my-flag'} />
+                                                ) : (
+                                                    <FeatureFlagInstructions
```

Comment:
I don't think we should get rid of `FeatureFlagInstructions` ever?

Reply:
personal opinion is that there should be a clear area for insights and a clear are for instrumenting/setup. This overview page is turning into an everything page

Reply:
Ah, so implementation instructions only on edit page? and insights on the view page?

Hmm, works.

Reply:
Missed that they're still on the edit/create page atleast

Reply:
Weird then though how the instructions are missing only when the flag is enabled?

I'd also propose having recordings on top of insights, as much more likely to have recordings (automatically generated, assuming enabled), than a manual insight using the flag.

Reply:
Non blocking though

Reply:
Missing when flag is enabled because I don't want to completely remove it yet. Leaving the original configuration without the flag in case users find it confusing